### ISSUE TITLE
[Bug]: Dimmer lights are broken after update to 0.6.0

### ISSUE THREAD

**Issue by johny-mnemonic** [JoDehli/PyLoxone#229]
### Describe the bug

After the update to 0.6.0 the dimmer lights are broken like this:
```
supported_color_modes:
  - b
  - e
  - g
  - h
  - i
  - "n"
  - r
  - s
  - s
  - t
color_mode: brightness
uuid: 1071d24e-0020-6a02-ffffd7846630eb23/AI2
room: ""
category: ""
device_typ: Dimmer
platform: loxone
max: 100
min: 0
device_class: Dimmer
icon: mdi:brightness-6
friendly_name: Noční světlo
supported_features: 0
```

After returning to 0.4.13 it gets fixed:
```
supported_color_modes:
  - brightness
color_mode: brightness
brightness: 63.75
uuid: 1071d24e-0020-6a02-ffffd7846630eb23/AI2
room: ""
category: ""
device_typ: Dimmer
platform: loxone
max: 100
min: 0
device_class: Dimmer
icon: mdi:brightness-6
friendly_name: Noční světlo
supported_features: 1
```

Seems like the parsing of `color_modes` doesn't work for some reason.

### Firmware of your Miniserver

14.4.9.25

### HomeAssistant install method

HA OS

### Version of HomeAssistant

2024.1.2

### Version of Pyloxone

0.6.0

### Update pyloxone

Return to 0.4.13 fixes the issue

### Log

Settings for the log can be found on the main page under the section Log Configuration

**Comment by JoDehli** [comment]
@johny-mnemonic how can I reproduce this issue. I get no errors on my system?

**Comment by johny-mnemonic** [comment]
I don't have any errors either. Just lights for which I am unable to control brightness.

I can try enabling the debug log if you think we might see something interesting there, but at first glance this looks like the code parsed something the wrong way, but is happy with the result = not logging errors.

**Comment by brechtvhb** [comment]
Got the same problem.

**Comment by ascha191** [comment]
same here

**Comment by elRadix** [comment]
> seems to be fixed in 0.6.1!

indeed it's fixed
